### PR TITLE
todo list peer-to-peer code review

### PR DESCRIPTION
**Pull request by DaveZag**
# Peer-to-peer todo list javascript code review.

### For this task :
- I created a main.js file 
- Put all the code from the modules in it.

### CODE REVIEW

**Comment by tashisrael** [comment]
## Hi @DaveZag , 
### I'm Tash, your PR reviewer

Good job so far! :boom:
There are some issues that you still need to work on but you are almost there!

## Highlights
- Descriptive README :heavy_check_mark: 
-  Good commit history :heavy_check_mark: 
- Linters are passing :heavy_check_mark: 
- Very professional README :heavy_check_mark: 
- Correct Github flow :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 
- Code follows DRY, KISS, and YAGNI rules :heavy_check_mark: 
- HTML & CSS best practices followed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- Kindly, make sure that you use ES6 syntax for all your functions
- Kindly, consider reducing the splitting of your code into smaller JavaScript files in the  modules folder for better readability.🙃

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @tashisrael in your question so I can receive the notification.

File: app/modules/checkboxes.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          return task;
+        });
+        localStorage.setItem('tasks', JSON.stringify(editedArr));
+      }
+    }
+  });
+}
+
+// check if checkbox is checked on load then style the checked task
+function checkOnLoad() {
```
Comment (tashisrael) [inline app/modules/checkboxes.js:44]: Kindly use Es6 syntax

File: app/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="icon"
+      href="./assets/favicon.png"
+      type="image/png"
+      sizes="16x16"
+    />
```
Comment (tashisrael) [inline app/index.html:14]: Kindly Indent properly

**Comment by DaveZag** [comment]
Thanks for the feedback.

**Comment by tashisrael** [comment]
Hi @DaveZag , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with refactoring the functions 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

File: app/index.html (hunk 2 above)
Comment (DaveZag) [reply app/index.html:14]: Hi, that's the way the code formatter was refactoring the code I don't understand why.🙂

**Pull request closed without merging**